### PR TITLE
Feature/update first time setup docs

### PR DESCRIPTION
**Pull request by lukavdplas**
Some clarification for the first-time setup documentation.

Notable changes:
- Added Node.js to prerequisites
- Removed instructions to enable CORS in elasticsearch, since I-analyzer does not make cross-origin requests
- Removed some duplicate instructions
- Moved instructions about configuring elasticsearch API key to settings documentation

### CODE REVIEW

File: documentation/Django-project-settings.md [hunk 1]
```diff
@@ -47,11 +47,17 @@ The values in the dictionary give specifications.
...
-- `'api_id'`
+By default, an elasticsearch server will have security features enabled; you can turn this off for a local development server (see [first-time setup](./First-time-setup.md)). Otherwise, the server configuration must specify an API key.
+
+Create an API key for the server: see [creating an API key](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html). Note down the `'id'` and `'api_key'` values of the response.
+
+Add the following values to the configuration:
+
+- `'certs_location'`: Fill in the following path: `{your_elasticsearch_directory}/config/certs/http_ca.crt`
+- `'api_key'`: the ID of the API key
+- `'api_id'`: the generated API key
```
Comment (JeltevanBoheemen) [inline documentation/Django-project-settings.md:60]: This seems the wrong way around

File: documentation/First-time-setup.md [hunk 2]
```diff
@@ -7,8 +7,9 @@ These are instructions to set up an I-analyzer server. If you are going to devel
 * Python == 3.9
 * PostgreSQL >= 10, client, server and C libraries
 * [ElasticSearch](https://www.elastic.co/) 8. To avoid a lot of errors, choose the option: install elasticsearch with .zip or .tar.gz. ES wil install everything in one folder, and not all over your machine, which happens with other options.
-* [Redis](https://www.redis.io/) (used by [Celery](http://www.celeryproject.org/)). Recommended installation is [installing from source](https://redis.io/docs/getting-started/installation/install-redis-from-source/)
-* Yarn
+* [Redis](https://www.redis.io/). Recommended installation is [installing from source](https://redis.io/docs/getting-started/installation/install-redis-from-source/)
+* [Node.js](https://nodejs.org/)
```
Comment (JeltevanBoheemen) [inline documentation/First-time-setup.md:11]: Specify Node version range?

Comment (JeltevanBoheemen) [reply documentation/First-time-setup.md:11]: Alternatively, refer to a `.nvmrc` file, easier to keep up-to-date in the source.

Comment (lukavdplas) [reply documentation/First-time-setup.md:11]: What versions do we support? The frontend README says node 12 but I don't think that's accurate.

Comment (JeltevanBoheemen) [reply documentation/First-time-setup.md:11]: https://angular.dev/reference/versions suggests `^12.20.0 || ^14.15.0 || ^16.10.0`

**Pull request merged**